playground/src/ShadowPlayground.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useRef } from 'react';
+import ReactDOM from 'react-dom';
+import { Playground } from './Playground';
+import './dockit-react-playground';
+import type { DockitReactPlayground } from './dockit-react-playground';
+
+type ShadowPlaygroundProps = {
+  /**
+   * The jsx to edit and render, as string.
+   */
+  code: string;
+
+  /**
+   * Specifies globals to use inside the live editor.
+   */
+  scope?: object;
+
+  /**
+   * Allows to render multiple components.
+   */
+  noInline?: boolean;
+
+  /**
+   * The styles to include in the shadow root.
+   */
+  styles?: string | CSSStyleSheet;
```

Comment:
`Props`+docgen didn't work with extended props implemented like this:

```
import { PlaygroundProps } from './Playground';

type ShadowPlaygroundProps = PlaygroundProps & {
  /**
   * The styles to include in the shadow root.
   */
  styles?: string | CSSStyleSheet;
};
```

it could only see new prop `styles`, but not original ones

so I had to define all props here leading to annoying duplication

on the other hand it works if I define manually `Playground.propTypes` & `ShadowPlayground.propTypes` (which is usually for JS users), but it does not avoid the duplication in my case, because then I still have TypeScript types  next to `Playground.propTypes` & `ShadowPlayground.propTypes`

I decided that this duplication is better, but ideally we should find the root cause and fix

Reply:
from Georges

> Docgen is not powerful enough. (And it's babel based that we want to replace). So don't waist too much time on props. You can just point to the doc of Playground for the time being.

so resolving this